### DISCUSSION TITLE
'PolicyInterpreter' object has no attribute 'decide'

### DISCUSSION THREAD

**Discussion by FatQiang** [DataCanvasIO/YLearn#39] [General]
When I run case_study_bank.ipynb, an error occur in line pi.decide(whatif_data) - 'PolicyInterpreter' object has no attribute 'decide'.

![image](https://user-images.githubusercontent.com/38529818/194453537-19a2b423-00b3-41cb-bb65-6fa86f978495.png)

**Comment by lixfz** [comment]
`PolicyInterpreter.decide()` was added in the main branch which is not released yet. Upgrade YLearn from  latest source code to run latest case study please.
